README.md

```diff
@@ -1,63 +1,57 @@
 
 ```
-Admin > User
+Student > Mara
+Student > Jeffrey
 
-allow User  /
-deny  User  /profile
-allow User  /profile/[id]
-allow Admin /profile
-deny  Admin /profile/*/password
-allow Admin /profile/[id]/password
+allow Admin   /
+allow Student /home/[id]
+allow Mara    /srv/nfs/music
+deny  Jeffrey /home/[id]/config
+deny  Admin   /home/*/personalsecrets
 ```
 
-These are the access rules for a website where people have a profile and everyone can access their own profile, but administrators can also access other people's profiles, except for the password sections.
+## The tools
 
-## Examples
-Here are some examples of valid sets of access rules.
+### Inheritance
+```
+A > B
+B > C
+
+allow A x
+deny  A x/*
+allow B x/y
+allow C x/z
+```
+`A` has access to `x`, `B` has access to both `x` and `x/y` and `C` has access to `x`, `x/y` and `x/z`.
 
-### Multiple inheritances
 ```
-B > A
-C > B
+A > B
 
-allow A a
-deny  A a/*
-allow B a/b
-allow C a/c
+allow A x/*
+deny  B x/y
 ```
-In this case, A has access to a, B has access to both a and a/b and C has access to a, a/b and a/c.
+Although `A` has access to `x` and `x/y`, `B` does not have access to `x/y` as inherited access is overridden.
 
 ### Sets
 ```
-Admin > User
+User > Admin
 
-allow User  devices
+allow Admin devices
 deny  User  devices/*
 allow User  devices/{ownedDevices}
 allow User  devices/{public}/control
 allow User  devices/{allowedDevices}/control
-allow Admin devices
-```
-In this case, a user has access to every device he owns and can control any device to which its owner has granted him access. The administrators however, have access to every device regardless of ownership.
-
-### Overriding inheritance
-```
-B > A
-
-allow A a
-allow A a/b
-deny  B a
 ```
-Although it has been specified that every A has access to a and a/b, since B does not have access to a, this overrides the aforementioned access to a and any lower level.
+A user has access to every device he owns and can control any device to which its owner has granted him access. The administrators however, have access to every device regardless of ownership.
 
 ### Multiple matching rules
 ```
-allow A a/*/c
-deny  A a/b/*
+allow A x/*/z
+deny  A x/y/*
 ```
-Both rules match the resource identifier a/b/c, but the identifiers matched by the first rule lie further apart than the identifiers matched by the second rule. Therefore, the second rule is more specific and will be applied, so A does not have access to a/b/c.
+Both rules match the resource identifier `x/y/z`. The identifier most specific on the highest level of difference in specificity takes precedence. Therefore, the second rule is more specific and `A` does not have access to `x/y/z`.
```

Comment:
I like your phrasing, but check the difference between an identifier and an (permission) rule.
Also, "the highest level of specicifity"?